packages/shared/src/types/firestore.ts

```diff
@@ -321,10 +330,11 @@ export interface EmailPayload {
 
 /**
  * A full email interface, including:
- * `to`, `from`, `subject`, `html` and `attachments`.
+ * `to`, `from`, `subject`, `html` and `attachments`. 'bcc'
  */
 export interface EmailMessage extends EmailPayload {
   from: string;
+  bcc: string;
```

Comment:
Can we call this `EmailPayload` since this is the `payload` property of process delivery document, and have the `EmailPayload` be renamed to `ClientEmailPaylad`??